AppleALC/ALCUserClient/ALCUserClient.cpp

```diff
@@ -0,0 +1,86 @@
+//
+//  ALCUserClient.cpp
+//  AppleALC
+//
+//  Created by Nick on 10/14/20.
+//  Copyright © 2020 vit9696. All rights reserved.
+//
+
+#include <IOKit/IOLib.h>
+
+#include "ALCUserClient.hpp"
+
+OSDefineMetaClassAndStructors(ALCUserClient, IOUserClient);
+
+const IOExternalMethodDispatch ALCUserClient::sMethods[kNumberOfMethods] = {
+	{ //kMethodExecuteVerb
+		(IOExternalMethodAction) &ALCUserClient::methodExecuteVerb,			// Method pointer
```

Comment:
I would use C++ casts here and everywhere else to match the rest of AppleALC code.

Reply:
Completely forgot. Fixed everywhere.